apps/client/pages/notice/index.page.tsx

```diff
@@ -23,7 +25,12 @@ const Notice = () => {
         <Text _fontSize={19} weight={weighs.bold} _color={colors.secondary[47]}>
           Notifications
         </Text>
-      )
+      ),
+      hideBackButton: true,
+      right: {
+        element: <Exit />,
+        onClick: () => back()
```

Comment:
```suggestion
        onClick: back
```

back 자체가 VoidFunction이라 이렇게 변경해도 잘 작동합니다!

Reply:
오 ! 감사합니다~!